src/app/api/notion/pages/route.ts

```
@@ -0,0 +1,99 @@
+  list: any[],
+  notion: Client
+) {
+  const children: ListBlockChildrenResponse = await notion.blocks.children.list(
+    {
+      block_id: pageId,
+    }
+  );
+  const childPages: any[] = await Promise.all(
+    children.results.map(async (child: any) => {
+      if (child.hasOwnProperty('child_page')) {
+        const subPage = await getChildPagesRecursive(child.id, list, notion);
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)